__init__.py

```diff
@@ -1,6 +1,8 @@
 """
 Home Assistant skill
 """  # pylint: disable=C0103
+from os.path import join as pth_join
```

Comment:
Optional - this might be a personal style thing - I'm not a big fan of importing packages as other names unless it's really needed eg you have namespace conflicts.

When you're reading the code and see `pth_join(args)` I would expect some locally defined function and assume it's doing something custom. Where as if I see `os.path.join()` I know exactly what it's going to do already.

Reply:
I do not like to import whole module If I just need one function but on other hand simple `join(args)` does not say much so I prefer to change naming to match original module. Would `path_join(args)` be better?

Reply:
happy with it as is - no stress :)